molr-commons/src/main/java/io/molr/commons/domain/MissionState.java

```diff
@@ -90,6 +95,18 @@ public RunState runState() {
         return strandRunStates.keySet();
     }
 
+    public Set<String> getBreakpointBlockIds() {
+        return this.breakpointBlockIds;
+    }
+    
+    public Map<String, Set<String>> getAllowedBlockCommandNamesById(){
```

Comment:
The problem with working with block objects instead of blockIds is, that we either need to transport the whole block information (id+text) within the MissionState or we need to accept that we would need the mission representation in order to deserialize an MissionStateDto to MissionState.

What do you think?